### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by timonbimon** [pytorch/vision#708]
Hi there, thanks for the awesome work, I use torchvision a lot :)

Was just wondering if there are any plans to make a new release and push it to PyPi soon? 
It's been 8 months since the last one and I feel like there have been quite a few improvements on the master branch. 

Cheers!

**Comment by hubutui** [comment]
Yeah, and there are a lot of new models, any plan pretrained models?

**Comment by fmassa** [comment]
Hi,

Yes, we will be cutting a new release soon, it's been a while!

@varunagrawal I'm thinking about making the `layers` be part of `0.3`, which would be the next release of torchvision.

**Comment by Oktai15** [comment]
@fmassa oh, it means that `layers` branch will be merged into master in nearest release?

It would be really great for any object detection task! Will wait :)

**Comment by fmassa** [comment]
Yes, we plan on having other domains as well outside of image classification for 0.3.

**Comment by TheCodez** [comment]
It would be nice if the GPU version of `NMS` from [maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/maskrcnn_benchmark/csrc/cuda/nms.cu) gets added as well.

**Comment by Oktai15** [comment]
@TheCodez `nms` is already in `layers', isn't it?

**Comment by TheCodez** [comment]
@Oktai15 only the CPU version is, the GPU version is not implemented yet.

**Comment by fmassa** [comment]
There is a CUDA implementation of `nms` in `maskrcnn-benchmark` that I plan to add to torchvision as well.

**Comment by varunagrawal** [comment]
@fmassa I can add the GPU `nms` over the weekend. In the meantime, can we work out a plan for the new release? Right now, the timelines seem kind of vague and that seems to be rustling feathers.

**Comment by varunagrawal** [comment]
@fmassa @TheCodez GPU version of NMS has been added to layers via #727

**Comment by stas00** [comment]
Another vote for new release, currently getting:
```
/home/stas/anaconda3/envs/fastai/lib/python3.7/site-packages/torchvision/models/squeezenet.py:94: UserWarning: nn.init.kaiming_uniform is now deprecated in favor of nn.init.kaiming_uniform_.
    init.kaiming_uniform(m.weight.data)
  /home/stas/anaconda3/envs/fastai/lib/python3.7/site-packages/torchvision/models/squeezenet.py:92: UserWarning: nn.init.normal is now deprecated in favor of nn.init.normal_.
    init.normal(m.weight.data, mean=0.0, std=0.01)
```
which is fixed in master. Please and thank you!

**Comment by fmassa** [comment]
@stas00 sure! I'm writing the release notes for the new release, sorry for the delay!

**Comment by surgan12** [comment]
@timonbimon this can be closed with the release of 0.2.3.
